cdap-docs/admin-manual/source/operations/monitoring.rst

```diff
@@ -17,8 +17,97 @@ See the :ref:`Logging <http-restful-api-logging>`, :ref:`Metrics <http-restful-a
 and :ref:`Monitoring <http-restful-api-monitor>` HTTP APIs, and the
 :ref:`Java Client <reference:java-client-api>` for more information.
 
+
 Monitoring Utilities
 ====================
 CDAP can be monitored using external systems such as `Nagios <https://www.nagios.org/>`__; a Nagios-style plugin 
 `is available <https://github.com/caskdata/cdap-monitoring-tools/blob/develop/nagios/README.rst>`__
 for checking the status of CDAP applications, programs, and the CDAP instance itself.
+
+
+.. _monitoring-metadata-update-notifications:
```

Comment:
It was in REST API documentation before. Since it is not REST API, we decided to move it to Administration section. But you are right, monitoring is not the right place.

I think the right place would be a new section called Metadata Update Notifications under Installation in Administration section. We can then have a link to it from the Metadata section in Developer section What do you say?